src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/cdi/ManagedBeanConstructorQuickFix.java

```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2021 IBM Corporation.
+*
+* This program and the accompanying materials are made available under the
+* terms of the Eclipse Public License v. 2.0 which is available at
+* http://www.eclipse.org/legal/epl-2.0.
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Contributors:
+*     Hani Damlaj
+*******************************************************************************/
+package io.openliberty.tools.intellij.lsp4jakarta.lsp4ij.cdi;
+
+import io.openliberty.tools.intellij.lsp4jakarta.lsp4ij.codeAction.proposal.quickfix.InsertAnnotationMissingQuickFix;
```

Comment:
Was there any problem with using the existing `io.openliberty.tools.intellij.lsp4mp4ij.psi.core.java.codeaction.InsertAnnotationMissingQuickFix` ?
I noticed the classes are slightly different.

Reply:
To use the lsp4mp quick fixes we would have to adopt their architecture. For instance they put some code in getCodeActions() and the rest of it in resolveCodeAction(). There is other code that decides whether to call the language server to resolve a code action or to call resolveCodeAction() without that. My thinking is not to change the architecture of the lsp4jakarta quick fixes for now. If you think the simpler implementation used by lsp4jakarta wont work with IntelliJ let's have a discussion.